### ISSUE TITLE
tsc.exe : Unsuported local

### ISSUE THREAD

**Issue by Bartolomeus-649** [Microsoft/TypeScript#9728]
When building in VS 2015 Update 3 (14.0.25422.01) I get this error:
error TS6049:Build:Unsupported locale 'sv-SE'.  (error MSB6006: "tsc.exe" exited with code 1.)

1. I did not specify this local
2. I don't want this local.
3. Stop localizing this kind of stuff since no one wants it, and you obviously can't get it to work.

**Build Output**
```
3>Task "VsTsc"
3>  C:\Program Files (x86)\Microsoft SDKs\TypeScript\1.8\tsc.exe  --module CommonJS --sourcemap --target ES5 --noEmitOnError --locale sv-SE "file1.ts" "file2.ts" ...
3>  C:\Code\Pri\Elitlamm\Main\Source\xxxx\error TS6049:Build:Unsupported locale 'sv-SE'.
3>C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\Microsoft.TypeScript.targets(242,5): error MSB6006: "tsc.exe" exited with code 1.
3>Done executing task "VsTsc" -- FAILED.

```

**Visual Studio Version info**
```
Microsoft Visual Studio Professional 2015
Version 14.0.25422.01 Update 3
Microsoft .NET Framework
Version 4.6.01038

Installed Version: Professional

LightSwitch for Visual Studio 2015   00322-40000-00000-AA915
Microsoft LightSwitch for Visual Studio 2015

Microsoft Visual Studio Tools for Applications 2015   00322-40000-00000-AA915
Microsoft Visual Studio Tools for Applications 2015

Windows Phone SDK 8.0 - ENU   00322-40000-00000-AA915
Windows Phone SDK 8.0 - ENU

Visual Basic 2015   00322-40000-00000-AA915
Microsoft Visual Basic 2015

Visual C# 2015   00322-40000-00000-AA915
Microsoft Visual C# 2015

Visual C++ 2015   00322-40000-00000-AA915
Microsoft Visual C++ 2015

Visual F# 2015   00322-40000-00000-AA915
Microsoft Visual F# 2015

Add New File   3.4
The fastest and easiest way to add new files to any project - including files that start with a dot

ASP.NET and Web Tools 2015 (RC1 Update 1)   14.1.20203.0
ASP.NET and Web Tools 2015 (RC1 Update 1)

ASP.NET Web Frameworks and Tools 2012.2   4.1.41102.0
For additional information, visit http://go.microsoft.com/fwlink/?LinkID=309563

ASP.NET Web Frameworks and Tools 2013   5.2.40314.0
For additional information, visit http://www.asp.net/

Azure App Service Tools v2.9   14.0.20316.0
Azure App Service Tools v2.9

Azure Data Lake Node   1.0
This package contains the Data Lake integration nodes for Server Explorer.

Azure Data Lake Tools for Visual Studio   2.0.6000.0
Microsoft Azure Data Lake Tools for Visual Studio

Bundler & Minifier   2.1.258
Adds support for bundling and minifying JavaScript, CSS and HTML files in any project.

Clang with Microsoft CodeGen   14.0.25317
Clang with Microsoft CodeGen

Common Azure Tools   1.8
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

DataFactoryProject   1.0
Microsoft Data Factory Package

Editor Enhancements   1.0.19
Provides additional features such as HTML and URL encodings, transformations and sorting of selected text the editor.

Fabric.ApplicationInsights   1.0
Allows user to instrument their Service Fabric projects for Application Insights.

Fabric.DiagnosticEvents   1.0
Fabric Diagnostic Events

File Nesting   2.5.62
Automatically nest files based on file name and enables developers to nest and unnest any file manually

GitHub.VisualStudio   1.0
A Visual Studio Extension that brings the GitHub Flow into Visual Studio.

Image Optimizer   3.6.89
Uses industry standard tools to optimize any JPEG, PNG and Gifs - including animated Gifs. Can do both lossy and lossless optimization.

Image Sprites   1.2.36
Boost your website's performance by creating image sprites to reduce the amount of HTTP requests needed.

JavaScript Language Service   2.0
JavaScript Language Service

JavaScript Project System   2.0
JavaScript Project System

KofePackagePackage Extension   1.0
KofePackagePackage Visual Studio Extension Detailed Info

Markdown Editor   1.5.110
A full featured Markdown editor with live preview and syntax highligting. Supports GitHub flavored Markdown.

Microsoft Azure Data Factory Node Node   1.0
Azure Data Factory extension for Visual Studio Server Explorer.

Microsoft Azure HDInsight HQL Service   2.0.6000.0
Language service for Hive query

Microsoft Azure HDInsight Tools for Visual Studio   2.0.6000.0
An integrated development environment for HDInsight application development.

Microsoft Azure Mobile Services Tools   1.4
Microsoft Azure Mobile Services Tools

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 2015 - v2.9.40323.3

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Team Foundation Server 2015 Power Tools   14.0
Power Tools that extend the Team Foundation Server integration with Visual Studio.

Microsoft Visual Studio Process Editor   1.0
Process Editor for Microsoft Visual Studio Team Foundation Server

NuGet Package Manager   3.4.4
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Office Developer Tools for Visual Studio 2015 ENU   14.0.23928
Microsoft Office Developer Tools for Visual Studio 2015 ENU

Open Command Line   2.1.175
Opens a command line at the root of the project. Support for all consoles such as CMD, PowerShell, Bash etc. Provides syntax highlighting, Intellisense and execution of .cmd and .bat files.

PreEmptive Analytics Visualizer   1.2
Microsoft Visual Studio extension to visualize aggregated summaries from the PreEmptive Analytics product.

Python Tools for Visual Studio   2.2.40623.00
Python Tools for Visual Studio provides IntelliSense, projects, templates, Interactive windows, and other support for Python developers.

Python Tools for Visual Studio - Django Integration   2.2.40623.00
Provides templates and integration for the Django web framework.

Python Tools for Visual Studio - Profiling Support   2.2.40623.00
Profiling support for Python projects.

SQL Server Analysis Services   13.0.1601.5
Microsoft SQL Server Analysis Services Designer 
Version 13.0.1601.5

SQL Server Data Tools   14.0.60525.0
Microsoft SQL Server Data Tools

SQL Server Integration Services   
Microsoft SQL Server Integration Services Designer
Version 13.0.1601.5

SQL Server Reporting Services   13.0.1601.5
Microsoft SQL Server Reporting Services Designers 
Version 13.0.1601.5

TFS Auto Shelve   5.4
Create a shelveset for the latest version of all pending changes, automatically.  Pending changes from the local TFS workspaces will be shelved to the TFS Server every "x" number of minutes when solutions are opened.

ToolWindowHostedEditor   1.0
Hosting json editor into a tool window

TypeScript   1.8.35.0
TypeScript tools for Visual Studio

Web Compiler   1.11.315
Compiler for LESS, Sass and CoffeeScript files

Web Essentials 2015.3   3.0.230
Adds many useful features to Visual Studio for web developers. Requires Visual Studio 2015

Visual C++ for Cross Platform Mobile Development (Android)   14.0.25401.00
Visual C++ for Cross Platform Mobile Development (Android)

Visual C++ for Cross Platform Mobile Development (iOS)   14.0.25401.00
Visual C++ for Cross Platform Mobile Development (iOS)

Visual Studio Spell Check Everywhere   VSSpellCheckEverywhere
An extension that enables spell checking within any Visual Studio file editor or tool window that uses WPF text boxes.
https://GitHub.com/EWSoftware/VSSpellChecker

Visual Studio Spell Checker   VSSpellChecker
An editor extension that checks the spelling of comments, strings, and plain text as you type or interactively with tool windows.
https://GitHub.com/EWSoftware/VSSpellChecker

Visual Studio Tools for Apache Cordova   Update 10
Visual Studio Tools for Apache Cordova

Visual Studio Tools for Universal Windows Apps   14.0.25420.01
The Visual Studio Tools for Universal Windows apps allow you to build a single universal app experience that can reach every device running Windows 10: phone, tablet, PC, and more. It includes the Microsoft Windows 10 Software Development Kit.

Workflow Manager Tools 1.0   1.0
This package contains the necessary Visual Studio integration components for Workflow Manager.

Xamarin   4.1.0.530 (2e39740)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin.Android   6.1.0.71 (4e27558)
Visual Studio extension to enable development for Xamarin.Android.

Xamarin.iOS   9.8.0.323 (39ebb77)
Visual Studio extension to enable development for Xamarin.iOS.

```

**Comment by Bartolomeus-649** [comment]
Temporary workaround

Modify the file
 `C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\TypeScript\Microsoft.TypeScript.targets`

by changing line 92
from this
`<PreferredUILang Condition="'$(BuildingInsideVisualStudio)' == 'true' and '$(PreferredUILang)' == ''">$([System.Globalization.CultureInfo]::CurrentUICulture.Name)</PreferredUILang>`

to this
`<PreferredUILang Condition="'$(BuildingInsideVisualStudio)' == 'true' and '$(PreferredUILang)' == ''"></PreferredUILang>`

**Comment by DickvdBrink** [comment]
For what it is worth it, I had the same issue with VS2015 Update 2, running on an English Windows 10 with NL language installed for typing.

**Comment by mhegazy** [comment]
this is the same as https://github.com/Microsoft/TypeScript/issues/8130. the issue has been fixed in TypeScript 2.0 and later.
